_posts/2019-04-07-sr2019-tlc-crowned-champion.md

```diff
@@ -0,0 +1,86 @@
+
+           
+{% include figure.html
+           class="right"
+           src="/images/content/news/sr2019/swi.jpg"
+           caption="Team SWI's volcano climbing robot in the Caldera" %}
+
+Our game this year, Caldera, challenged teams to control scoring zones with tokens while navigating an arena with raised platforms. Twenty-five scoring zones were arranged in a grid, varying in value from two points for each of the outermost 16 zones, a ring of eight on a raised “volcano” worth seven points apiece, to the the central “caldera” which was worth thirty points.
+
+There were a range of inventive designs this year. Many teams opted for tank tracks as a method of climbing the volcano. Instead of climbing the volcano, a few teams built catapult mechanisms with varying degrees of success. Others scooped up tokens and deposited them on the low scoring areas and/or the edge of the volcano. Some teams lifted tokens onto top-mounted conveyor belts which dropped them onto the volcano's surface.
+
+Some of the more notable designs were:
+
+- Team TLC with their consistent, successful catapult.
+- Team SWI with their simple and effective grab-and-go robot which climbed the volcano with a single token.
+- Team CRB with their suction cup and catapult combination which unfortunately had technical issues for much of the competition.
+- Team HAB with their Hawaiian themed robot, topped with a 3D paper pineapple.
+
+The Final
+---------
+
+Student Robotics 2019 concluded with an intense match between PSC, HAM, TLC, and HRS. Team HAM were quick off the mark pushing their tokens into the scoring zones. Team HRS soon followed, violently shoving their tokens across the arena. Team PSC's robot slowly moved 4 of their tokens into a single scoring zone before unfortunately flipping onto its back. Meanwhile, Team TLC quietly moved towards the edge of the volcano and launched a token directly into the Caldera.
+
+<!-- Video of Final -->
+<iframe
+  class="center video"
+  src="https://www.youtube.com/embed/gOeP96OngE0"
+  frameborder="0"
+  width="100%"
+  allowfullscreen
+  >
+</iframe>
+
+We saw some brilliant robot designs this year, great work by all teams. We hope you enjoyed the competition as much as we did!
+
+If you didn't take part, or you want to enter again next year, the [sign up page](/schools/how_to_enter) for next year's competition will be up later in the year. Get a team together and start talking to your teachers now!
+
+Results
+-------
+
+|        Prize          |            Team
+|-----------------------|-----------------------------------------------
+| 1st Place             | [The Ladies' College](http://www.ladiescollege.com/)
+| 2nd Place             | [Hills Road Sixth Form College](http://www.hillsroad.ac.uk/)
+| 3rd Place             | [Hampton School](https://hamptonschool.org.uk/) and [Lady Eleanor Holles School](https://www.lehs.org.uk/)
+| Rookie Award          | [Eltham College](https://www.elthamcollege.london/)
```

Comment:
Just a thought -- could we use title text on the names of the prizes to include the descriptions of the awards in the page?

I suspect that something like the following would do this:
``` markdown
<span title="The team which eats the most chocolate">Award name</span> | Team Name Here
```